### ISSUE TITLE
no method matching keymap(::Vector{Dict}}

### ISSUE THREAD

**Issue by JeffreySarnoff** [chakravala/Reduce.jl#46]
```

julia> using Reduce
[ Info: Precompiling Reduce [93e0c654-6965-5f22-aba9-9c1ae6b3c259]
ERROR: InitError: MethodError: no method matching keymap(::Vector{Dict})
Closest candidates are:
  keymap(::Any, ::Union{REPL.LineEdit.HistoryPrompt, REPL.LineEdit.PrefixHistoryPrompt}) at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.6\REPL\src\LineEdit.jl:2007
  keymap(::Union{Vector{Dict{Any, Any}}, Vector{Dict{Char, Any}}}) at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.6\REPL\src\LineEdit.jl:1610
  keymap(::REPL.LineEdit.PromptState, ::REPL.LineEdit.Prompt) at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.6\REPL\src\LineEdit.jl:2505
  ...
Stacktrace:
 [1] initrepl(parser::typeof(Reduce.parserepl); prompt_text::String, prompt_color::Symbol, start_key::Char, repl::REPL.LineEditREPL, mode_name::String, valid_input_checker::Function, keymap::Dict{Char, Any}, completion_provider::REPL.REPLCompletionProvider, sticky_mode::Bool, startup_text::Bool)
   @ ReplMaker ~\.julia\packages\ReplMaker\pwo5w\src\ReplMaker.jl:78
 [2] repl_init(repl::REPL.LineEditREPL)
   @ Reduce ~\.julia\packages\Reduce\1n3pM\src\repl.jl:45
 [3] repl_init
   @ ~\.julia\packages\Reduce\1n3pM\src\repl.jl:60 [inlined]
 [4] __init__()
   @ Reduce ~\.julia\packages\Reduce\1n3pM\src\Reduce.jl:322
 [5] _include_from_serialized(path::String, depmods::Vector{Any})
   @ Base .\loading.jl:597
 [6] _require_from_serialized(path::String, cache::Base.TOMLCache)
   @ Base .\loading.jl:649
 [7] _require(pkg::Base.PkgId, cache::Base.TOMLCache)
   @ Base .\loading.jl:952
 [8] require(uuidkey::Base.PkgId, cache::Base.TOMLCache)
   @ Base .\loading.jl:836
 [9] require(into::Module, mod::Symbol)
   @ Base .\loading.jl:824
during initialization of module Reduce
```
```
julia> versioninfo()
Julia Version 1.6.0-DEV.913
Commit 4c805d2310 (2020-09-14 14:07 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Xeon(R) E-2176M  CPU @ 2.70GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-9.0.1 (ORCJIT, skylake)
Environment:
  JULIA_NUM_THREADS = 8
```

**Comment by chakravala** [comment]
That's an issue with [ReplMaker](https://github.com/MasonProtter/ReplMaker.jl) in Julia 1.6 not `Reduce` so you should transfer that issue over there if you want it to get fixed, since the fix is not going to be in this repository.

**Comment by JeffreySarnoff** [comment]
Thank you for the clarification.  I am raising the issue there.

**Comment by MasonProtter** [comment]
The bugfix should be available in ReplMaker 0.2.4 which is now merged in the general registry.
